hs_linux/storage.py

```diff
@@ -0,0 +1,140 @@
+from django.core.files.storage import  FileSystemStorage
+from hs_linux.models import LinuxAVU
+import os, tarfile, shutil, errno
+from pprint import pprint
+
+IRODS_PATH = "./irods/"
```

Comment:
Why is there an IRODS named variable in the LinuxStorage implementation?  This looks like a root path for configuring a linux storage location on disk.